tests/run/pep563_annotations.py

```diff
@@ -3,7 +3,7 @@
 
 from __future__ import annotations
 
-def f(a: 1+2==3, b: list, c: this_cant_evaluate, d: "Hello from inside a string") -> "Return me!":
+def f(a: 1+2==3, b: list, c: this_cant_evaluate, d: Hello from inside a string) -> Return me!:
```

Comment:
And this file - now invalid syntax. This is specifically testing various annotations so annotations need to remain the same